tools/devctr/poetry.lock

```diff
@@ -2,87 +2,87 @@
 
 [[package]]
 name = "aiohttp"
-version = "3.9.2"
+version = "3.9.1"
```

Comment:
why this version is getting downgraded?

Reply:
Mh, good catch, I might have messed up a merge conflict in the lock file :thinking:

Reply:
Fixed (its now showing an update because the first step of building a new docker container is "update poetry dependencies")!

Reply:
thanks for the change